code/modules/clothing/modular_armor/attachments/storage.dm

```diff
@@ -18,10 +18,14 @@
 	var/show_storage = FALSE
 	///Icon for the extra storage overlays.
 	var/show_storage_icon = 'icons/mob/modular/modular_helmet_storage.dmi'
+	///If our module is capable of tactical reload with a gun
+	var/allows_tactical_reload
```

Comment:
Why not just set this to true instead of setting it to true manually on every subtype?

Reply:
Because I want the var on storage level without having to set it to false on every single storage that disallows it.

Reply:
I suggest setting it to be explicitly FALSE instead of null at least, then. For clarity.